lib/src/lib/components/form-fields/select-field/select-field.component.ts

```diff
@@ -79,6 +86,7 @@ export class SelectFieldComponent
 
     this.addSubscription(
       this.optionsFilter$.pipe(
+        debounceTime(this.options?.search?.debounceTime ?? 0),
```

Comment:
Something to discuss, but i would not default it to 0, default should have a debounce of 350ms for example.

Reply:
added 275ms as the default